src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,94 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../components/Loader';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import PersonItem from '../components/PersonItem';
+
+const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+
+  useEffect(() => {
+    if (!isError) {
```

Comment:
The condition `if (!isError)` inside the `useEffect` hook might cause the fetch operation to be retried immediately after an error, due to the `setIsError(false)` call inside the `catch` block. Consider revising the logic to prevent unnecessary re-fetching, such as by using a separate state to control retries.